packages/common/src/utils/wallet.ts

```diff
@@ -151,11 +151,13 @@ export const ceilingBNUSDCToNearestCent = (value: BNUSDC): BNUSDC => {
 /** Formats a USDC wei string (full precision) to a fixed string suitable for
 display as a dollar amount. Note: will lose precision by rounding _up_ to nearest cent */
 export const formatUSDCWeiToUSDString = (amount: StringUSDC, precision = 2) => {
+  // remove negative sign if present.
+  const amountPos = amount.replace('-', '')
```

Comment:
This was a nice little hack for the AUDIO tx history, but it feels ripe for bugs in other potential (future?) usages... we should probably revisit both of these at some point.